core/src/apps/cosmos/transaction.py

```
@@ -309,6 +299,45 @@ def __init__(self):
         self.i18n_value = "MsgMultiSend"
 
 
+class NobleVaultTxn:
+    DECIMAL_PLACES = 6
+    TOKEN_SYMBOL = "USDN"
```

Ruff (0.11.9)

Possible hardcoded password assigned to: "TOKEN_SYMBOL"

(S105)